be/database/migrations/2023_07_21_023120_create_host_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('host', function (Blueprint $table) {
+            $table->id();
+            $table->string('first_name');
+            $table->string('last_name');
+            $table->string('user_name');
+            $table->string('password');
+            $table->string('emails');
+            $table->integer('contact_num');
```

Comment:
It is much better to make this as string since we cannot put `0` in front of the number
```suggestion
            $table->string('contact_num');
```

Reply:
okay sir